sources/packages/api/src/services/institution/institution.service.ts

```diff
@@ -16,11 +25,45 @@ export class InstitutionService extends RecordDataModelService<Institution> {
   constructor(
     @Inject("Connection") connection: Connection,
     private readonly bceidService: BCeIDService,
+    private readonly userService: UserService,
   ) {
     super(connection.getRepository(Institution));
     this.logger.log("[Created]");
   }
 
+  toDtoObject(institutionEntity: Institution): InstituteDto {
```

Comment:
I would consider moving this logic to the InstituteDto and have it be responsible for creating the Dto from a Institution object.
In the future, if every dto have its own logic, it will prevent us from adding all to conversion to the service layer.
I would also be considering adding a library like the below one to do the mappings to us. It will handle the majority of the heavy working by conversion and allow us to configure what is not resolved by convention.
https://github.com/loedeman/AutoMapper/wiki/Getting-started

Reply:
👍